src/graphorm/index.ts

```diff
@@ -199,6 +199,22 @@ const obj = new GraphORM({
           return value;
         },
       },
+      collectionSources: {
+        relation: {
+          isMany: true,
+          customRelation: (ctx, parentAlias, childAlias, qb): QueryBuilder => {
+            return qb
+              .where(
+                `${childAlias}.id = ANY(${parentAlias}."collectionSources")`,
```

Comment:
Why do we need the `ANY` here?

Reply:
`collectionSources` is array of source ids

Reply:
Got it :)